.github/workflows/test.yml

```
@@ -190,7 +190,7 @@ jobs:
   test-stateful-sync:
     needs: build
     uses: ./.github/workflows/gcp-test-deploy.yml
-    if: ${{ github.event.inputs.regenerate-disks != 'true' }}
+    if: ${{ github.event.inputs.regenerate-disks != 'true' || github.event.inputs.run-full-sync != 'true' }}
```

We only want to run these tests when we are not running any manual tests

```suggestion
    if: ${{ github.event.inputs.regenerate-disks != 'true' && github.event.inputs.run-full-sync != 'true' }}
```